Go/Go.sublime-syntax

```diff
@@ -967,175 +1059,269 @@ contexts:
     - include: pop-type
 
   pop-parameter-list-unnamed:
-    - match: \)
-      scope: punctuation.section.parens.end.go
-      pop: true
-    - include: match-comments
-    - include: match-keywords
+    - include: pop-on-paren-end
     - include: match-comma
     - include: match-ellipsis
+    - include: match-keywords
     - match: (?=\S)
       push: pop-type
```

Comment:
The way `match-type` is implemented the following would work, too, and avoid one extra context switch.

```suggestion
    - include: match-type
```